test/lit/passes/optimize-instructions-gc.wast

```diff
@@ -238,17 +238,13 @@
   ;; CHECK-NEXT:   )
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT:  (drop
-  ;; CHECK-NEXT:   (i32.eqz
-  ;; CHECK-NEXT:    (ref.is_null
-  ;; CHECK-NEXT:     (local.get $func)
-  ;; CHECK-NEXT:    )
+  ;; CHECK-NEXT:   (ref.is_func
+  ;; CHECK-NEXT:    (local.get $func)
```

Comment:
Why is this now less optimized?

Reply:
A PR before this sounds good, just so we don't have a regression in the middle (could affect bisection in the future, in principle). I could do it if you want as I assume I wrote the original.

Reply:
Actually, this has significant overlap with your ongoing changes to `ref.cast` optimization, so it might be better to wait and reuse the code we end up with there.

Reply:
Ok, fair enough, let's not block on that then, and leave it.